tools/macos_buildenv.sh

```diff
@@ -119,6 +108,9 @@ case "$1" in
             echo "Exported environment variables:"
             echo_exported_variables
         fi
+
+        brew install ccache
+        brew install make # is installed as gmake
```

Comment:
This script should be able to be run without Homebrew installed. I think it would be better to have these in the GitHub Actions workflow file. Also, if we're installing a make program that doesn't come with the OS, let's use Ninja.